### PR TITLE
[Jetsnack] Use lazy row/columns.

### PR DESCRIPTION
**Pull request by nickbutcher**

### CODE REVIEW

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt [hunk 1]
```diff
@@ -47,55 +46,62 @@ import com.example.jetsnack.ui.components.JetsnackSurface
...
+        snackCollections,
+        onSnackClick,
+        modifier
+    )
+}
+
+@Composable
+private fun Feed(
+    filters: List<Filter>,
+    snackCollections: List<SnackCollection>,
```
Comment (manuelvicnt) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt:70]: Nit: Feels like passing `snackCollections` as an argument first and then `filters` reads better when calling this function.

**Review by manuelvicnt — APPROVED** [review]
Very small nit, LGTM anyway :D

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt (hunk 1 above)
Comment (chrisbanes) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt:70]: +1

**Review by JoseAlcerreca — APPROVED** [review]

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt (hunk 1 above)
Comment (nickbutcher) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt:70]: Yeah, agreed.

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**